Framework/Crystal/test/PredictSatellitePeaksTest.h

```diff
@@ -193,4 +195,87 @@ class PredictSatellitePeaksTest : public CxxTest::TestSuite {
 
     AnalysisDataService::Instance().remove("peaks");
   }
+
+  void test_exec_multiple_goniometers_lean() {
+    /* create a TOPAZ instrument */
+    FrameworkManager::Instance().exec("LoadEmptyInstrument", 4, "InstrumentName", "TOPAZ", "OutputWorkspace", "inst");
+
+    /* Add it to a Peaks workspace */
+    TS_ASSERT_THROWS_NOTHING(FrameworkManager::Instance().exec("CreatePeaksWorkspace", 8, "InstrumentWorkspace", "inst",
```

Comment:
And remove `"InstrumentWorkspace", "inst"`

Reply:
done!